website/client/assets/svg/positive.svg

```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="10" height="10" viewBox="0 0 10 10">
-    <path fill-rule="evenodd" d="M6 4V0H4v4H0v2h4v4h2V6h4V4H6z"/>
```

Comment:
so there's a small problem here because if you set the fill parameter directly on the SVG, it's impossible to override it. So what we've been doing, when an icon is used in different colors, is to remove the fill parameter from the .svg file and specify the color as a css rule for the `.svg-icon` element

Reply:
Okay, so I should make a css rule for .svg-icon in profile.vue then?

Reply:
Yes, in this case since there are multiple icons you'll want to use a more specific class like `.remove-icon`,  ...